packages/ui-components/src/chat/message.tsx

```diff
@@ -169,7 +174,12 @@ function MessageComponent(props: MessageProps): JSX.Element {
                     </EditButtons>
                 </div>
             )}
-            {!editMode && <MessageBody>{localMessage.message}</MessageBody>}
+            {!editMode && (
+                <MessageBody>
+                    {localMessage.message}
+                    {localMessage.edited && <EditedText> (edited)</EditedText>}
```

Comment:
I think it would look better if the `(edited)` indicator was part of the timestamp

Reply:
Have added to the top as well, which do you think is better?

<img width="296" alt="Screenshot 2024-04-02 at 14 34 22" src="https://github.com/causalens/dara-ui/assets/104359539/790b55f7-da44-4acb-b027-11e052162b83">

Reply:
I also like it in the timestamp :) this is how whatsapp does it

Reply:
Slack does at the end, it seems to vary quite widely where people add it. But will go for timestamp since we have two votes for it :)